src/cloudAdapter/genezio/genezioAdapter.ts

```diff
@@ -29,11 +29,11 @@ import { UserError } from "../../errors.js";
 import { stdout } from "process";
 import { createHash } from "../../utils/strings.js";
 
-const BUNDLE_SIZE_LIMIT = 256901120;
+const BUNDLE_SIZE_LIMIT = 1.024e9;
 async function handleBigElementSizeError(
     element: GenezioCloudInput,
     projectConfiguration: ProjectConfiguration,
-    BUNDLE_SIZE_LIMIT: number = 256901120,
+    BUNDLE_SIZE_LIMIT: number = 1.024e9,
```

Comment:
This approach will also affect classes and functions - I doubt we want that (but maybe it does not affect us that much).

And a nit - below the error throws a hardcoded message `The maximum size is 250MB.` Update that also - maybe even use `BUNDLE_SIZE_LIMIT` instead of hardcoded